### DISCUSSION TITLE
Brew linux distrubition not working

### DISCUSSION THREAD

**Discussion by wimdeblauwe** [jreleaser/jreleaser#1593] [Q&A]
Hi,

I a have a BINARY distrubution configured (See https://github.com/wimdeblauwe/ttcli/blob/main/pom.xml#L223) and I also have brew configured with multiplatform. Although I have a linux BINARY configured, the binary is not available for Linux. I received a PR to fix this on the linked `homebrew-ttcli` repo (https://github.com/wimdeblauwe/homebrew-ttcli/pull/2/files), but I guess this should be fixed in my JReleaser configuration. How can I add linux support as well?

**Comment by aalmiray** [comment]
The current behavior of the brew package is to only accept `.zip` files. The linux binary produces by your project has `.tar.gz`.

Workaround1: use `.zip` for the linux artifact. 

Workaround2: generate a custom templare that adds support for the linux artifact as `.tar.gz`.

Moving forward, JReleaser could certainly support more extensions other than `.zip` but that'll likely break some builds if not guarded (multiple matching artifacts for a given packager). Thus, adding this behavior in deactivated mode by default is the way to go for the time being. A configuration flag would be needed to activated it. This behavior would then be switched on for `2.x` when we expect to break compatibility. 

WDYT?

**Reply by wimdeblauwe** [reply]
I'll use the `.zip` artifact, no problem. Thanks!

**Reply by aalmiray** [reply]
You can find a list of supported artifact extensions at the packager's reference page https://jreleaser.org/guide/latest/reference/packagers/homebrew.html

I believe `.zip` was chosen because I could not find examples of non `.zip` artifacts when brew support was first added. This decision was not revisited when multi-platform support was added.

**Reply by aalmiray** [reply]
Filed https://github.com/jreleaser/jreleaser/issues/1594
